lib/compare_task.rb

```diff
@@ -74,17 +74,18 @@ def render_comparison(description1, description2, scopes, options = {})
         partial_description_common = SystemDescription.new(
           "common",
           store,
-          scope => comparison[2]
+          scope => comparison[3]
         )
 
         output += Renderer.for(scope).render_comparison(
           partial_description1,
           partial_description2,
+          comparison[2],
```

Comment:
This is inconsistent in terms of the API. It would be more logical to also wrap the changed elements in a description, so that the call would use the arguments `partial_description1, partial_description2, partial_description_changed, partial_description_common`.